0613 fluid

sarahlshowers opened the pull request.

sarahlshowers committed:
fluid header/footer

sarahlshowers committed:
header and footer updated to be fully fulid and responsive

sarahlshowers committed:
adding inner footer to wrap for max-width styling for extra large viewports
header and footer tested on iPad2 and 13" displays

sarahlshowers committed:
styling for smallest viewports, iPhone4 - iPhone6+ (portrait)
tested on iPhone4s and iPhone6
also tested for firefox, safari and chrome support

derrickshowers commented on app/wp-content/themes/kstg/library/scss/components/_footer.scss:
```diff
@@ -42,95 +35,91 @@ footer {
-      li:first-child {
-        border-left: 1px solid #CCCCCC;
-      }
+      // li:first-child {
```
Can this just be removed if you don't need it?

derrickshowers commented on app/wp-content/themes/kstg/library/scss/components/_footer.scss:
```diff
@@ -42,95 +35,91 @@ footer {
     ul {
       li {
-        border-right: 1px solid #CCCCCC;
+        // border-right: 1px solid #CCCCCC;
```
Can this just be removed if you don't need it?

derrickshowers commented on app/wp-content/themes/kstg/library/scss/components/_footer.scss:
```diff
@@ -1,8 +1,11 @@
+  }
+  height: 63px;
   bottom: 0;
+  margin-top: -16px;
```
I may be missing something here, but I would avoid a negative margin if you can. One, it's just one of those things to avoid where possible. More importantly, you may have some issues on your other pages if your footer is creeping up into the section above it, forcing you to always have to put extra padding on what's above. 

I would just modify the padding on the `.aside-wrapper` class (probably in the homepage scss?)

derrickshowers commented on app/wp-content/themes/kstg/library/style.css:
```diff
@@ -0,0 +1,25 @@
+/*
```
Something isn't right here. Looks like there was an error when the css was being compiled. Not sure how things are working?

sarahlshowers committed:
deleted extra style.css, not sure where it came from and it's removal doesn't seem to affect the page
removed commented out styling
left negative margin until all branches are merged and I can see what's going on
